src/backend/looped/weapons/aimbot.cpp

```diff
@@ -55,98 +60,98 @@ namespace big
+						Vector3 multiply                 = cam_direction * distance;
+						Vector3 front_cam                = cam_coords + multiply;
+						camera_target                    = front_cam - CAM::GET_GAMEPLAY_CAM_COORD();
+						g_aimbot.smooth_factor           = camera_target;
+						g_aimbot.using_aimbot_first_time = false;
+					}
+					Vector3 target = g_aimbot.aim_lock - CAM::GET_GAMEPLAY_CAM_COORD();
+					g_aimbot.smooth_factor.x += (target.x - g_aimbot.smooth_factor.x) * g.weapons.aimbot.smoothing_speed / 10.f;
+					g_aimbot.smooth_factor.y += (target.y - g_aimbot.smooth_factor.y) * g.weapons.aimbot.smoothing_speed / 10.f;
+					g_aimbot.smooth_factor.z += (target.z - g_aimbot.smooth_factor.z) * g.weapons.aimbot.smoothing_speed / 10.f;
 
-				std::chrono::duration<double> elapsed_time = std::chrono::system_clock::now() - current_time;
-				if (elapsed_time.count() > 0.f)
+					camera_target = g_aimbot.smooth_factor;
+				}
+				else
 				{
-					INPUT mouse_handle; // MOUSEINPUT mi;
-					mouse_handle.type = INPUT_MOUSE;
-					mouse_handle.mi.dwFlags = MOUSEEVENTF_MOVE; // Type = Mouse movement, and the event  is emulating the mouse movement
+					camera_target = g_aimbot.aim_lock - CAM::GET_GAMEPLAY_CAM_COORD();
+				}
+				if (g_aimbot.aim_lock.x == 0.f && g_aimbot.aim_lock.y == 0.f && g_aimbot.aim_lock.z == 0.f)
+					return;
 
-					// Update the mouse by moving it with how much we need / smoothing speed
-					mouse_handle.mi.dx = mouse_movement.x / (g.weapons.aimbot.smoothing ? g.weapons.aimbot.smoothing_speed : 2);
-					mouse_handle.mi.dy = mouse_movement.y / (g.weapons.aimbot.smoothing ? g.weapons.aimbot.smoothing_speed : 2);
-					SendInput(1, &mouse_handle, sizeof(mouse_handle)); //handles the input
+				float camera_heading = atan2f(camera_target.x, camera_target.y) * 180.0f / (float)M_PI;
+				float magnitude      = sqrtf(camera_target.x * camera_target.x + camera_target.y * camera_target.y
+                    + camera_target.z * camera_target.z);
 
-					//Reset our variables
-					mouse_movement.x = 0, mouse_movement.y = 0;
-					update_time_now = true; //reset our time
+				float camera_pitch = asinf(camera_target.z / magnitude) * 180.0f / (float)M_PI;
+				float self_heading = ENTITY::GET_ENTITY_HEADING(self::ped);
+				float self_pitch   = ENTITY::GET_ENTITY_PITCH(self::ped);
+				if (camera_heading >= 0.0f && camera_heading <= 180.0f)
+				{
+					camera_heading = 360.0f - camera_heading;
+				}
+				else if (camera_heading <= -0.0f && camera_heading >= -180.0f)
+				{
+					camera_heading = -camera_heading;
+				}
+				if (CAM::GET_FOLLOW_PED_CAM_VIEW_MODE() == 4)
+				{
+					CAM::SET_FIRST_PERSON_SHOOTER_CAMERA_HEADING(camera_heading - self_heading);
+					CAM::SET_FIRST_PERSON_SHOOTER_CAMERA_PITCH(camera_pitch - self_pitch);
+				}
+				else
+				{
+					CAM::SET_GAMEPLAY_CAM_RELATIVE_HEADING(camera_heading - self_heading);
+					CAM::SET_GAMEPLAY_CAM_RELATIVE_PITCH(camera_pitch - self_pitch, 1065353216);
```

Comment:
Why are you giving 1065353216 as the parameter? Did you copy this from somewhere? I don't know where, but it's probably somewhere that doesn't understand floating point numbers (scripts?). 1065353216 is the decimal representation of 0x3F800000 which is the IEEE 754 representation of 1.0f. Did you test this?